client/rpcserver/handlers.go

```diff
@@ -9,13 +9,27 @@ import (
 	"sort"
 	"strings"
 
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/dex"
 	"decred.org/dcrdex/dex/msgjson"
 )
 
+// routes
 const (
+	closeWalletRoute = "closewallet"
 	helpRoute        = "help"
-	versionRoute     = "version"
+	initRoute        = "init"
+	newWalletRoute   = "newwallet"
+	openWalletRoute  = "openwallet"
 	preRegisterRoute = "preregister"
+	versionRoute     = "version"
+	walletsRoute     = "wallets"
+)
+
+const (
+	INITIALIZED     = "initialized"
+	WALLET_LOCKED   = "wallet locked"
+	WALLET_UNLOCKED = "wallet unlocked"
```

Comment:
While not explicitly forbidden, upper-case constants are not common in go, and underscores [are discouraged](https://golang.org/doc/effective_go.html#mixed-caps).